quickfixj-core/pom.xml

```diff
@@ -59,7 +59,7 @@
         <dependency>
             <groupId>org.apache.mina</groupId>
             <artifactId>mina-core</artifactId>
-            <version>2.1.6</version>
+            <version>2.2.2-SNAPSHOT</version>
```

Comment:
I had this artifact in my local repository so the build here will fail.

Reply:
Should we wait for 2.2.2 release or will 2.2.1 do?

Reply:
2.2.1 will not work, it doesn't have endpoint identification algorithm for SNI.